### PR TITLE
Add `card-social-reply`

### LINKED ISSUE #58: [Request] card-social-reply
https://dribbble.com/shots/6384317/attachments/1367003?mode=media

image

### PR DESCRIPTION
**Pull request by bossthanawat**
closes #58

### CODE REVIEW

**Comment by siriwatknp** [comment]
@all-contributors please add @bossthanawat for code

File: blocks/card-social-reply/src/CardSocialReply.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import {
+  Avatar,
+  Card,
+  CardContent,
+  Sheet,
+  Stack,
+  styled,
+  Typography,
+} from "@mui/joy";
```
Comment (siriwatknp) [inline blocks/card-social-reply/src/CardSocialReply.tsx:10]: Please use direct component import for the convention:

```js
import Avatar from '@mui/joy/Avatar';
…other import
```

Comment (siriwatknp) [reply blocks/card-social-reply/src/CardSocialReply.tsx:10]: `styled` can be imported from `'@mui/joy/styles';`

Thanks!

File: blocks/card-social-reply/src/CardSocialReply.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                >
+                  <Avatar
+                    alt="avatar1"
+                    src="https://mui.com/static/images/avatar/3.jpg"
+                    sx={{ width: 24, height: 24 }}
+                  />
+                  <Typography
+                    level="title-sm"
+                    fontWeight="lg"
+                    sx={{ whiteSpace: "nowrap" }}
```
Comment (siriwatknp) [inline blocks/card-social-reply/src/CardSocialReply.tsx:77]:
```suggestion
                    noWrap
```
There is a `noWrap` prop for the Typography, can you try?

File: blocks/card-social-reply/src/CardSocialReply.tsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                  </Typography>
+
+                  <Typography
+                    level="body-xs"
+                    sx={{
+                      whiteSpace: "nowrap",
+                      overflow: "hidden",
+                      textOverflow: "ellipsis",
+                      minWidth: 0,
+                    }}
```
Comment (siriwatknp) [inline blocks/card-social-reply/src/CardSocialReply.tsx:89]:
```suggestion
                    noWrap
```

**Review by siriwatknp — COMMENTED** [review]
👍 Great first PR! Thanks for the help

File: blocks/card-social-reply/src/CardSocialReply.tsx (hunk 2 above)
Comment (bossthanawat) [reply blocks/card-social-reply/src/CardSocialReply.tsx:75]: I think the ui wants to show the full text

If I use noWrap
image

Comment (siriwatknp) [reply blocks/card-social-reply/src/CardSocialReply.tsx:75]: Got it.

**Pull request merged**